### PR TITLE
PMM-3668 build ami image for PMM2

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: ansible/pmm2/main.yml [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+         - percona-clickhouse-server
+         - pmm2-client
+         - unzip
+       state: installed
+       enablerepo: "pmm-laboratory"
+
+    - name: PMM                        | Prometheus METRICS_RETENTION
+      command: sh -c 'sed -i "s/ENV_METRICS_RETENTION/${METRICS_RETENTION:-720h}/" /etc/supervisord.d/pmm.ini'
+
+    - name: PMM                        | Disable crond
```
Comment (delgod) [inline ansible/pmm2/main.yml]:
```suggestion
    - name: PMM                        | Disable clickhouse-server in systemd
```

File: ansible/pmm2/main.yml [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    - name: NGINX SSL Certificate      | Generate certificate
+      when: not certificate_file.stat.exists
+      command: /var/lib/cloud/scripts/per-boot/generate-ssl-certificate
+
+    - name: Grafana                    | Check data dir
+      stat: path=/srv/grafana/grafana.db
+      register: grafana_db
+
+    - name: PMM                        | Fix path to grafana.db
+      command: sh -c 'sed -i "s|/var/lib/grafana/grafana.db|/srv/grafana/grafana.db|" /etc/supervisord.d/pmm.ini'
```
Comment (delgod) [inline ansible/pmm2/main.yml]: why not fix it in pmm-server repo directly?

File: ansible/pmm2/main.yml (hunk 1 above)
Comment (hors) [reply ansible/pmm2/main.yml]: Thank you!

File: ansible/pmm2/main.yml (hunk 2 above)
Comment (hors) [reply ansible/pmm2/main.yml]: I will create a separate task for these changes and as soon as it is fixed will change this code.

**Pull request merged**